Demo/Common/IntegrationExamples/Components/CardComponentAdvancedFlowExample.swift

```diff
@@ -125,8 +126,12 @@ internal final class CardComponentAdvancedFlowExample: InitialDataAdvancedFlowPr
             self.dismissAndShowAlert(success, message)
         }
     }
+    
+    private func presentAlert(with error: Error, retryHandler: (() -> Void)? = nil) {
+        presenter?.presentAlert(with: error, retryHandler: retryHandler)
+    }
 
-    internal func dismissAndShowAlert(_ success: Bool, _ message: String) {
+    private func dismissAndShowAlert(_ success: Bool, _ message: String) {
```

Comment:
Great with the private changes! 
We may realize these are internal because of the tests on the next tasks :D but let's see